### ISSUE TITLE
Assign individual aircraft to flights

### ISSUE THREAD

**Issue by lorlandi** [nabeelio/phpvms#858]
The ability to assign a flight to a specific aircraft.

**Comment by lorlandi** [comment]
In order to create flights assigned to a specific aircraft we can use the same concept of "subfleet"
For example:
In our aircraft database we have 22 planes and 22 subfleets, the following is an example that shows that each subfleet has a certain aircraft as this example:

![sbfletts](https://user-images.githubusercontent.com/4042146/95640571-b6d71d80-0a73-11eb-83c3-4c5d615a55bd.jpg)

In the case of the 737Freighter fleet, each subflleet on this class has its own aircraft:
![B737F](https://user-images.githubusercontent.com/4042146/95640464-4e883c00-0a73-11eb-9e59-aa61fd9bd27f.jpg)
![B737F_1](https://user-images.githubusercontent.com/4042146/95640466-51832c80-0a73-11eb-8d79-b086904ed3e1.jpg)
![B737F_2](https://user-images.githubusercontent.com/4042146/95640467-52b45980-0a73-11eb-8a95-0cf9f56cd199.jpg)
And the same way in all subfleet.

The "aircraft" database looks like this:
![aircraftt_data_base](https://user-images.githubusercontent.com/4042146/95640673-349b2900-0a74-11eb-98c1-1b9ae7e093fd.jpg)

In the following picture "flight_subfleet" database we can see that the first flight correspond to subfleet_id "11" which is an B752 register N364TP of the "aircraft" table 
![flight_subfleet](https://user-images.githubusercontent.com/4042146/95640832-141f9e80-0a75-11eb-9ba1-c27c05257807.jpg)

Continuing down on the table, appears the subfleet_id “3” that corresponds to the B737 register CC-BTP .
![flight_subfleet1](https://user-images.githubusercontent.com/4042146/95640967-f1da5080-0a75-11eb-9c71-78f063a76919.jpg)

These examples show that the structures of the databases do not change, no alterations to the code or changes to the databases were made.
The only thing that has been done is to use the benefits of the system and follow the instructions given regarding the creation of a subfleet "You can create as many as you want, you need at least one"
And in this way we have a flight assigned to a specific plane.
------------------------------------------------------------------------------------------------------------------------------------
ACARS System:
Now we will see what happens when a flight is booked (bid) and you want to load it into PHPVMS ACARS, for example a bid of flight TPR110-A
![my_bid](https://user-images.githubusercontent.com/4042146/95641239-8c875f00-0a77-11eb-80e2-40e5c279873f.jpg)

Now we load our flight into the ACARS system
![Acars](https://user-images.githubusercontent.com/4042146/95641345-159e9600-0a78-11eb-81d8-0716df5582aa.jpg)

Flight is loaded, now we select the aircraft that we will use for the flight:
![Acars1](https://user-images.githubusercontent.com/4042146/95641372-441c7100-0a78-11eb-934d-dad3a37b8b8e.jpg)

When choosing the aircraft, they appear according to the aircrafts that the pilot's rank allows to use and not by subfleet, and as we saw previously, the flight_subfleet table already has the flights with a designated aircraft.
The "Aircraft" field should load the subfleet assigned to that flight and not the list of aircraft by pilot's rank.

------------------------------------------------------------------------------------------------------------------------------------

In conclusion, the system has the necessary characteristics to enable the designation of a flight to a specific aircraft. Some improvements could be implemented

- Aircraft field in ACARS load the subfleet assigned to that flight according to data in flight_subfleet.
- In the page "Flights" add the aircraft assigned to the flight.

I hope this explanation to this "feature request" has been clear.
Thanks and regards

**Comment by lorlandi** [comment]
With the last ACARS version 1.0.0.1132 this "feature request" works excellent.
Also adding this code on flights/table.blade we can see the subfleet name

@foreach($flight->subfleets as $subfleet) --> **AFTER @foreach($flights as $flight)**
{{ $subfleet->name }}
@endforeach --> **AFTER the last @endforeach**

For example in the flights page:

            <div class="col-sm-5">
                @if($flight->route)
                    <b style="font-weight: 900;">{{ strtoupper(__('flights.route')) }} -</b>
                    {{ $flight->route }}
                @endif
            </div>
			
		<div class="col-sm-5">
                            <b style="font-weight: 900;">{{ strtoupper(__('subfleets')) }} -</b>
                            {{ $subfleet->name }}
                 </div>

**Comment by lorlandi** [comment]
Thanks again Nabeel.
